modules/flowable-engine/src/main/java/org/flowable/engine/impl/cmd/JobRetryCmd.java

```diff
@@ -160,6 +157,13 @@ public Object execute(CommandContext commandContext) {
         return null;
     }
 
+	protected static String getFailedJobRetryTimeCycleValue(FlowElement currentFlowElement) {
+        if (currentFlowElement instanceof ServiceTask) {
+            return ((ServiceTask) currentFlowElement).getFailedJobRetryTimeCycleValue();
+        }
+		return null;
+	}
```

Comment:
Please format this code block so it aligns with surrounding code and matches the project's formatting.  Thanks.

Reply:
Done, thanks :)